gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,134 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
+
 ----
 
+.. _v24.3.0:
+
+24.3.0
+======
+
+.. _v24.3.0-new-features:
+
+New Features
+------------
+
+
+* Added support to enable NVIDIA GDRCopy v2.4.1.
+
+  When you enable support for GDRCopy, the Operator configures the GPUCopy Driver container image
+  as a sidecar container in the GPU driver pod.
+  The sidecar container compiles and installs the gdrdrv Linux kernel module.
+  This feature is supported on Ubuntu 22.04 and RHCOS operating systems and on X86_64 and ARM64 architectures.
+
+  Refer to :ref:`Chart Customization Options` for more information about the ``driver.gdrcopy`` field.
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.54.15.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit version v1.15.0
+    - NVIDIA MIG Manager version v0.6.0
+    - NVIDIA Driver Manager for K8s v0.6.8
+    - NVIDIA Kubernetes Device Plugin v0.15.0
+    - DCGM 3.3.3-1
+    - DCGM Exporter 3.3.5-3.4.0
+    - Node Feature Discovery v0.15.4
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.15.0
+    - NVIDIA MIG Manager for Kubernetes v0.6.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.6
+    - NVIDIA vGPU Device Manager v0.2.5
+
+* Added support for Kubernetes v1.29 and v1.30.
+  Refer to :ref:`Supported Operating Systems and Kubernetes Platforms`.
+
+* Added support for NVIDIA GH200 Grace Hopper Superchip as a generally available feature.
+  Refer to :ref:`supported nvidia gpus and systems`.
+
+  The following prerequisites are required for using the Operator with GH200:
+
+  - Run Ubuntu 22.04, the 550.54.15 GPU driver, and an NVIDIA Linux kernel, such as one provided with a ``linux-nvidia-<x.x>`` package.
+  - Add ``init_on_alloc=0`` and ``memhp_default_state=online_movable`` as Linux kernel boot parameters.
+  - Run the NVIDIA Open GPU Kernel module driver.
+
+* Removed support for Kubernetes pod security policy (PSP).
+  PSP was deprecated in the Kubernetes v1.21 release and removed in v1.25.
+
+.. _v24.3.0-fixed-issues:
+
+Fixed Issues
+------------
+
+* Installation on Red Hat OpenShift Container Platform 4.15 no longer requires a workaround related to
+  secrets and storage for the integrated image registry.
+* Previously, the vGPU Device Manager encountered an error if no NVIDIA devices were found in ``/sys/class/mdev_bus``.
```

Comment:
```suggestion
* Previously, the vGPU Device Manager would panic if no NVIDIA devices were found in ``/sys/class/mdev_bus``.
```

Reply:
I wasn't sure if that was too technical, but I suppose admins have read that in logs before.  Made the change, TY.